microsoft-365/managed-desktop/service-description/DeviceRegistration.md

```diff
@@ -125,38 +138,19 @@ You can monitor the progress of device registration on the main **Microsoft Mana
 
-<table>
-    <tr>
-        <td>Error Message</td>
-        <td>Details</td>
-    </tr>
-    <tr>
-        <td>Unexpected Error</td>
-        <td>Your request could not be automatically processed. Please contact Support <link to Ops page> and provide the Request ID from your error export.</td>
-    </tr>
-    <tr>
-        <td>Invalid Hardware Hash</td>
-        <td>The hardware hash you provided for this device was incorrectly formatted. Please verify the hardware hash <link to docs> and resubmit. </td>
-    </tr>
-    <tr>
-        <td>Device already registered</td>
-        <td>This device is already registered to your company. No further action required.</td>
-    </tr>
-    <tr>
-        <td>Device not found</td>
-        <td>We couldn’t deregister this device because it does not exist in your company. No further action required. </td>
-    </tr>
-    <tr>
-        <td>Device claimed by another company</td>
-        <td>This device has already been claimed by another company. Please check with your device supplier. </td>
-    </tr>
-    <tr>
-        <td>Device not found</td>
-        <td>We couldn’t register this device because we could not find a match for the provided Manufacturer, Model, and Serial Number. Please confirm the values with your device supplier.</td>
-    </tr>
-    </table>
+| Error | Details |
+|----------------------|---------------------|
+| Unexpected Error |Your request could not be automatically processed. Contact Support [link to Ops page] and provide the Request ID from your error export. |
+| Invalid Hardware Hash | The hardware hash you provided for this device was not formatted correctly. Double-check the hardware hash (see [Prepare to register devices](#prepare-to-register-devices)) and then resubmit. |
+|Device already registered |This device is already registered to your company. No further action required. |
+| Device not found |We couldn’t de-register this device because it does not exist in your company. No further action required. |
+| Device claimed by another company | This device has already been claimed by another company. Check with your device supplier. |
+| Device not found | We couldn’t register this device because we could not find a match for the provided manufacturer, model, or serial number. Confirm these values with your device supplier. |
 
+[//]: # (A few notes/questions: we can't use HTML tables. We don't say "please" except in very particular circumstances. Would it be possible to change "Invalid hardware hash" to "Hardware hash not valid"? [we avoid "invalid"]. Earlier we heavily implied that the man./model didn't have to match exactly, but here it seems it'll throw an error if not...so what's the story there? Is "de-register" a typo--i.e., should it be "register?" Or is there some kind of de-registration action that we just haven't discussed yet? Oh, do we explain anywhere how to export errors, since we ask for that in the first row of the table?)
 
```

Comment:
RE: HTML - rgr. I'm good with MD format from now on.

RE: not saying please, I'm good with that.

RE: Hardware Hash invalid - good with your edit

RE: model/man - that error message should only come up for Partners (see below). If we have a separate topic, maybe we remove it from this table since users should never see it.

RE: yeah, there's a de-registration action. However, we recently decided not to allow customers to do that on their own in v1. We can remove those rows from the table for now (and chat more about deregistration offline).

RE: explaining export - we should add that! Totally a miss.

Reply:
"Invalid": I meant in the error itself, which of course would be code change. The doc of course will quote whatever the UI or error string says. The "please" thing is actually MS policy--it's a globalization/cultural thing.

Reply:
hey @Magana, can you integrate the feedback on the error text for hardware hash?

Reply:
Sure thing!

Reply:
Sweet, @jaimeo - let me know what you think about having a separate table for PArtners. Also, let's remove the de-registration rows completely for now.

Reply:
Sorry, thought I mentioned that: yes of course since we'll have a completely separate Partners topic, that topic will have its own version of the table. I think even if there is some duplication of the errors in the table(s), it's worth doing b/c once we vector a reader down a particular track, it's best to keep them on that track w/o bouncing over to topics on the other track, if you see what I mean.